libraries/AP_NavEKF3/AP_NavEKF3_Measurements.cpp

```diff
@@ -506,7 +506,7 @@ void NavEKF3_core::readGpsData()
     const AP_GPS &gps = AP::gps();
 
     if (gps.last_message_time_ms() - lastTimeGpsReceived_ms > frontend->sensorIntervalMin_ms) {
-        if (gps.status() >= AP_GPS::GPS_OK_FIX_3D) {
+        if (gps.status() >= gps.status_arm_min()) {
```

Comment:
same comment here as in the EKF2: we should accept data below GPS status arm min

Reply:
Done